src/pages/creations/index.tsx

```diff
@@ -32,7 +49,7 @@ export default function Creations() {
             <HeroDivider>
               <CreationsTitleContainer>
                 <SquigglyTitle
-                  text='WHAT’S COOKING?'
+                  text='MADE IN WONDERLAND'
                   sizes={{
```

Comment:
**Just a quick note:** the current version of the creations hero looks like this:  
![image](https://github.com/user-attachments/assets/bd1169d0-dcbd-4de0-8fc8-588004e8e554)

Mati and I were reviewing it, and on his computer, the "Made in Wonderland" text appears **huge**:  
![image](https://github.com/user-attachments/assets/6337f6b6-436d-4ccc-862d-2f88ee7d4d57)

On my end, it looks more like this:  
![image](https://github.com/user-attachments/assets/ffd260bc-f77f-4059-9bd4-361043c0244b)

Could we consider reducing the text size for different screen resolutions? Not sure

Reply:
https://github.com/defi-wonderland/web/pull/114/commits/bd489c9aa3a93b955c137ca49e16d9083e8521ad